nengo_spinnaker/simulator.py

```diff
@@ -47,6 +48,9 @@ def __init__(self, network, dt=0.001, period=10.0):
             Duration of one period of the simulator. This determines how much
             memory will be allocated to store precomputed and probed data.
         """
+        # Register `close` to be called when the interpreter is closed.
+        atexit.register(self.close)
```

Comment:
Registering this close function will stop Python from ever actually deleting any Simulator object, since there's always going to be a reference to it, even if it has been manually closed.  It might be better to just add one static atexit handler and keep a static list of unclosed Simulators.